pages/linux/winetricks.md

```diff
@@ -0,0 +1,16 @@
+# winetricks
+
+> Manage virtual Windows environments.
```

Comment:
```suggestion
> Manage Wine virtual Windows environments.
```

Reply:
I think that's a lot better than what reads in `man`: `winetricks - manage virtual Windows environments using Wine`

The "using Wine" is really vague.